Modules/System/Permission Sets/src/TenantPermissionSubForm.page.al

```diff
@@ -375,6 +644,46 @@ page 9859 "Tenant Permission Subform"
         CurrPage.Update(false);
     end;
 
+    local procedure UpdateSelected(RIMDX: Text[1]; PermissionOption: Option)
```

Comment:
wouldn't it be better to move this code into a permission codeunit?
Or at least to code inside the loop?

Reply:
Makes sense. Will do. This function is a copy of the old function that Microsoft created.

Reply:
done